manifests/4.5/logging.openshift.io_clusterlogforwarders_crd.yaml

```diff
@@ -9,7 +9,7 @@ spec:
     listKind: ClusterLogForwarderList
     plural: clusterlogforwarders
     singular: clusterlogforwarder
-  scope: cluster
+  scope: Namespaced
```

Comment:
we have scope as `Namespaced` here but in the manifest we are missing a namespace for the metadata

Reply:
Excellent point, need to fix the manifest.

Reply:
Will fix.